bot/tasks/pennychat.py

```diff
@@ -233,6 +266,41 @@ def _penny_chat_details_blocks(penny_chat_invitation, mode=None):
     return body
 
 
+def _followup_reminder_blocks(penny_chat_invitation):
+    organizer = get_or_create_social_profile_from_slack_id(
+        penny_chat_invitation.organizer_slack_id,
+        slack_client=get_slack_client(),
+    )
+    return [
+        {
+            'type': 'section',
+            'text': {
+                'type': 'mrkdwn',
+                'text': f'{organizer.real_name}\'s Penny Chat *"{penny_chat_invitation.title}"* has completed. '
+                        'How did it go?\n\nIf you learned something new, then consider writing a Penny Chat '
+                        'Follow-Up and sharing what you learned with the community. This is also a great way '
+                        'to say _*thank you*_ to all those that attended the Penny Chat.'
+            }
+        },
+        {
+            'type': 'actions',
+            'elements': [
+                {
+                    'type': 'button',
+                    'text': {
+                        'type': 'plain_text',
+                        'text': ':pencil2: Write a Follow-Up',
+                        'emoji': True
+                    },
+                    'style': 'primary',
+                    'url': f'https://www.pennyuniversity.org/chats/{penny_chat_invitation.id}',
```

Comment:
we probably need to make `https://www.pennyuniversity.org/` and environment var. What should we call it.